src/yunohost/user.py

```diff
@@ -453,3 +441,10 @@ def _convertSize(num, suffix=''):
             return "%3.1f%s%s" % (num, unit, suffix)
         num /= 1024.0
     return "%.1f%s%s" % (num, 'Yi', suffix)
+
+
+def _hash_user_password(password):
+    char_set = string.ascii_uppercase + string.ascii_lowercase + string.digits + "./"
+    salt = ''.join([random.SystemRandom().choice(char_set) for x in range(16)])
```

Comment:
Well, we can point out to python official doc if you want yes.

Red square on the page https://docs.python.org/2/library/random.html or https://docs.python.org/2/library/random.html#random.SystemRandom but red square is better I think